reference/3.0/Microsoft.PowerShell.Management/Checkpoint-Computer.md

```diff
@@ -7,40 +7,49 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=135197
 external help file:  Microsoft.PowerShell.Commands.Management.dll-Help.xml
 title:  Checkpoint-Computer
 ---
-
 # Checkpoint-Computer
+
 ## SYNOPSIS
+
```

Comment:
Sorry, we need to change the rules in your script for `## SYNOPSIS`. We can't have a blank line after the H2 for this one header.